src/editors/containers/TextEditor/components/ImageUploadModal.jsx

```diff
@@ -1,44 +1,67 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+
+import tinyMCEKeys from '../../../data/constants/tinyMCE';
 import ImageSettingsModal from './ImageSettingsModal';
 import SelectImageModal from './SelectImageModal';
 import * as module from './ImageUploadModal';
 
+export const propsString = (props) => Object.keys(props)
+  .map(key => `${key}="${props[key]}"`)
+  .join(' ');
+
+export const imgProps = ({ settings, selection }) => ({
```

Comment:
nit of the most nitpicky (of work that I think is partially mine lol): Is it potentially misleading to call something which is not React  a `prop` instead of an `attribute`

Reply:
eh, I wouldn't say so.  Is a react app and we are building UI html references.  I would almost rather we be using JSX with a react method to use JSX here rather than a flat string.

```
const htmlString = ReactDOMServer.renderToStaticMarkup(
      <div ...
);
```

Reply:
Not saying we have to do it but just want to follow your thinking:  Would the advantage of using `renderToStaticMarkup` that it would be more readable because jsx gives us nice non-string format and strings are harder to read than code? Or is there a performance// maintainability thing here that i'm missing?

Reply:
this is less verifyable.  Is more "read this string... does it look right?" whereas jsx has built-in validation that we are making a valid html element